kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/KcatClient.java

```diff
@@ -71,9 +72,13 @@ public void produceMessages(String topicName, String bootstrap, String message,
 
         LOGGER.atInfo().setMessage("Producing messages in '{}' topic using kcat").addArgument(topicName).log();
         String name = Constants.KAFKA_PRODUCER_CLIENT_LABEL + "-kcat-" + TestUtils.getRandomPodNameSuffix();
+        String jsonOverrides = getInstance().isOpenshift() ? TestUtils.getJsonFileContent("nonJVMClient_overrides.json") : "";
+
         List<String> executableCommand = new ArrayList<>(List.of(cmdKubeClient(deployNamespace).toString(), "run", "-i",
                 "-n", deployNamespace, name,
                 "--image=" + Constants.KCAT_CLIENT_IMAGE,
+                "--override-type=strategic",
+                "--overrides=" + jsonOverrides.replace("%NAME%", name),
```

Comment:
we should probably do the replace on line 75 and only in the `isOpenshift` branch.